### PR TITLE
Add octavia operator

### PR DESCRIPTION
**Pull request by adrianfusco**
Generating configuration for octavia operator

### CODE REVIEW

**Comment by adrianfusco** [comment]
@Sandeepyadav93 can you take a look? Thank you

File: ci-operator/config/openstack-k8s-operators/octavia-operator/openstack-k8s-operators-octavia-operator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- as: golint
+  commands: make golint
+  container:
+    from: src
+- as: gofmt
+  commands: make gofmt || true
+  container:
+    from: src
+zz_generated_metadata:
+  branch: master
```
Comment (Sandeepyadav93) [inline ci-operator/config/openstack-k8s-operators/octavia-operator/openstack-k8s-operators-octavia-operator-master.yaml:46]: Octavia-operator has `main` branch.

master need to be replaced by `main`. Make changes in this file and run `make jobs` command.

Comment (adrianfusco) [reply ci-operator/config/openstack-k8s-operators/octavia-operator/openstack-k8s-operators-octavia-operator-master.yaml]: Thank you, Sandeep. I've changed the references to main and executed the command. It seems that the jobs have been triggered.

**Comment by Sandeepyadav93** [comment]
Waiting for Tom Weining to join `openshift` org so that he can be added in OWNERS file.

File: ci-operator/config/openstack-k8s-operators/octavia-operator/OWNERS [hunk 2]
```diff
@@ -0,0 +1,16 @@
+approvers:
+- abays
+- dprince
+- olliewalsh
```
Comment (Sandeepyadav93) [inline ci-operator/config/openstack-k8s-operators/octavia-operator/OWNERS:4]: Please add weinimo and beagles in OWNERS file.

File: ci-operator/config/openstack-k8s-operators/octavia-operator/openstack-k8s-operators-octavia-operator-main.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.46.0
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
```
Comment (Sandeepyadav93) [inline ci-operator/config/openstack-k8s-operators/octavia-operator/openstack-k8s-operators-octavia-operator-main.yaml:11]: Could we please update this tag to: rhel-8-release-golang-1.18-openshift-4.12

**Review by Sandeepyadav93 — COMMENTED** [review]
Imho.. It would be good to squash the changes in a single commit to have a cleaner git history.

File: ci-operator/config/openstack-k8s-operators/octavia-operator/openstack-k8s-operators-octavia-operator-main.yaml (hunk 3 above)
Comment (adrianfusco) [reply ci-operator/config/openstack-k8s-operators/octavia-operator/openstack-k8s-operators-octavia-operator-main.yaml]: Changed. Thanks

File: ci-operator/config/openstack-k8s-operators/octavia-operator/OWNERS (hunk 2 above)
Comment (adrianfusco) [reply ci-operator/config/openstack-k8s-operators/octavia-operator/OWNERS:4]: Added. Thanks.

**Pull request merged**